Automatic Validation ended with:
> 2024-03-06T00:35:35.6291365Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][0] Description: Missing required property 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: PeterPawlowski.foobar2000.EncoderPack.installer.yaml Manifest Error: Required field missing. [InstallerSha256]

(Automated response - build 861.)

--- a/manifests/p/PeterPawlowski/foobar2000/EncoderPack/2024-03-03/PeterPawlowski.foobar2000.EncoderPack.installer.yaml
+++ b/manifests/p/PeterPawlowski/foobar2000/EncoderPack/2024-03-03/PeterPawlowski.foobar2000.EncoderPack.installer.yaml
@@ -7,5 +7,6 @@ InstallerType: nullsoft
 Installers:
 - Architecture: x86
   InstallerUrl: https://www.foobar2000.org/getfile/Free_Encoder_Pack-2024-03-03.exe
+  InstallerSha256: DEBDAF4C9BFC6DB3B364278F1366A500D730C505531FEB6D96AC1BE1AD9BDA92
 ManifestType: installer
 ManifestVersion: 1.6.0

--- a/manifests/p/PeterPawlowski/foobar2000/EncoderPack/2024-03-03/PeterPawlowski.foobar2000.EncoderPack.installer.yaml
+++ b/manifests/p/PeterPawlowski/foobar2000/EncoderPack/2024-03-03/PeterPawlowski.foobar2000.EncoderPack.installer.yaml
@@ -6,7 +6,7 @@ PackageVersion: 2024-03-03
 InstallerType: nullsoft
 Installers:
 - Architecture: x86
-  InstallerUrl: https://www.foobar2000.org/getfile/Free_Encoder_Pack-2024-03-03.exe
-  InstallerSha256: DEBDAF4C9BFC6DB3B364278F1366A500D730C505531FEB6D96AC1BE1AD9BDA92
+  InstallerUrl: https://www.foobar2000.org/files/Free_Encoder_Pack-2024-03-03.exe
+  InstallerSha256: a64e67ec304979e831efa2bfe6925042a773200c7d6153eeec71fcce1ce0bf10
 ManifestType: installer
 ManifestVersion: 1.6.0

